src/main/java/homeworks/jose/secondAssigment/ClassA.java

```diff
@@ -0,0 +1,15 @@
+package homeworks.jose.secondAssigment;
+
+public class ClassA extends Father{
+    private List<> c;
+
+    @Override
+    public void methodB() {
+        super.methodB();
+    }
+
+    @Override
+    public void methodC() {
+        super.methodC();
```

Comment:
there is no need for an override (and therefore the method) since the behavior of both methods are the same